16_wordle/main.go

```diff
@@ -99,3 +99,46 @@ func checkChar(chosenWordSlice []string, inputSlice []string) []string {
 	}
 	return containedSlice
 }
+
+func checkCharPrecise(chosenWordSlice []string, inputSlice []string) []string {
+	var containedSlice []string
+	// 1 = contained but incorrect loc
+	// 2 = contained in correc loc
+	// 0 not contained
+
+	// correct word check
+	if slices.Equal(chosenWordSlice, inputSlice) {
+		return []string{"Correct word!"}
+	}
```

Comment:
This is inconsistent with the expected output.
If you wanted to keep []string, you could do:

```suggestion
	if slices.Equal(chosenWordSlice, inputSlice) {
		return []string{"22222"}
	}
```